packages/wrangler/package.json

```diff
@@ -38,6 +38,7 @@
   "dependencies": {
     "@esbuild-plugins/node-globals-polyfill": "^0.1.1",
     "@esbuild-plugins/node-modules-polyfill": "^0.1.4",
+    "@webcontainer/env": "^1.0.1",
```

Comment:
could you move this to devDependencies please? We bundle all the js, so it doesn't have to be installed as a dep.

Reply:
Yep for sure!